st2tests/integration/orquesta/test_wiring.py

```diff
@@ -144,3 +147,19 @@ def test_output_on_error(self):
 
         self.assertEqual(ex.status, ac_const.LIVEACTION_STATUS_FAILED)
         self.assertDictEqual(ex.result, expected_result)
+
+    def test_config_context_renders(self):
+        packs = ["orquesta_tests", "dummy_pack_7"]
```

Comment:
These are packs for unit tests.  We have separate ones for integration tests.  I use the examples pack.  But there are other packs that st2 CI uses.  Looking at how the st2 environment is setup in CI and reusing existing test packs and installing those test packs on setup is simpler than writing an entire InstallFixturesPacks class here.

Reply:
> These are packs for unit tests. We have separate ones for integration tests. I use the examples pack. But there are other packs that st2 CI uses. Looking at how the st2 environment is setup in CI and reusing existing test packs and installing those test packs on setup is simpler than writing an entire InstallFixturesPacks class here.

Submitted code. Want to make sure I am in the right track. Did look at existing pack to replace dummy_pack_7 and still did not get the right one. I think we should create a new pack in the case integration test like this that needs too packs  with complicated setup. Thanks!